EDDI/MainWindow.xaml

```diff
@@ -12,20 +12,23 @@
         mc:Ignorable="d"
         Title="EDDI" ResizeMode="CanResizeWithGrip" Width="928" Height="600" MinWidth="928" MinHeight="600">
     <Window.Resources>
+        <!-- Colors and simple brushes -->
+        <Color x:Key="DeepPurple" A="0xff" R="0x43" G="0x35" B="0x5d"/>
+        <SolidColorBrush x:Key="NeutralBackgroundBrush" Color="{x:Static SystemColors.ControlLightColor}"/>
         <SolidColorBrush x:Key="DataGridLineBrush" Color="LightGray"/>
+        <!-- Gradients -->
+        <LinearGradientBrush x:Key="DockPanelBackgroundBrush" EndPoint="0.5,1.0" MappingMode="RelativeToBoundingBox" StartPoint="0.5,0.0">
+            <GradientStop Color="Black" Offset="0.0"/>
+            <GradientStop Color="{StaticResource DeepPurple}" Offset="1.0"/>
+        </LinearGradientBrush>
+        <!-- Styles -->
         <Style TargetType="DataGrid">
             <Setter Property="AlternatingRowBackground" Value="AliceBlue"/>
             <Setter Property="HorizontalGridLinesBrush" Value="{StaticResource DataGridLineBrush}"/>
             <Setter Property="VerticalGridLinesBrush" Value="{StaticResource DataGridLineBrush}"/>
         </Style>
     </Window.Resources>
-    <DockPanel LastChildFill="True">
-        <DockPanel.Background>
-            <LinearGradientBrush EndPoint="0.5,1" MappingMode="RelativeToBoundingBox" StartPoint="0.5,0">
-                <GradientStop Color="Black"/>
-                <GradientStop Color="#FF4E355D" Offset="1"/>
-            </LinearGradientBrush>
-        </DockPanel.Background>
+    <DockPanel LastChildFill="True" Background="{StaticResource DockPanelBackgroundBrush}">
```

Comment:
I considered making the background implicit via a `Style` object but ultimately rejected it. I think it's the right approach to make all DataGrids look the same, but this background is rather custom.